app/lib/i18n/fi.i18n.json

```diff
@@ -1,394 +1,390 @@
 {
```

Comment:
We shouldn't really have any changes to this file, as it managed by our localization tool.